AutoRunCustomTool.sln

```diff
@@ -1,22 +1,48 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26014.0
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.32112.339
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoRunCustomTool", "AutoRunCustomTool\AutoRunCustomTool.csproj", "{B61D18B8-CDC7-4D5F-A82B-EE6231126ECA}"
 EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "AutoRunCustomToolShared", "AutoRunCustomToolShared\AutoRunCustomToolShared.shproj", "{8E0C7164-B1EB-4C6B-8FC2-7D43AB8DDDC2}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoRunCustomTool64", "AutoRunCustomTool64\AutoRunCustomTool64.csproj", "{7A559F3B-752A-4FF6-A29E-E299FC9A8F59}"
+EndProject
 Global
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		AutoRunCustomToolShared\AutoRunCustomToolShared.projitems*{7a559f3b-752a-4ff6-a29e-e299fc9a8f59}*SharedItemsImports = 4
+		AutoRunCustomToolShared\AutoRunCustomToolShared.projitems*{8e0c7164-b1eb-4c6b-8fc2-7d43ab8dddc2}*SharedItemsImports = 13
+		AutoRunCustomToolShared\AutoRunCustomToolShared.projitems*{b61d18b8-cdc7-4d5f-a82b-ee6231126eca}*SharedItemsImports = 4
+	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|x86 = Release|x86
```

Comment:
Is there a reason to add this new configuration? Both projects are built for the AnyCPU platform. I don't think the x86 configuration is needed.

Reply:
You are right. I will remove the x86 configuration